spring-kafka/src/test/java/org/springframework/kafka/retrytopic/RetryTopicIntegrationTests.java

```diff
@@ -138,6 +143,17 @@ void shouldRetryFourthTopicWithNoDlt() {
 		assertThat(awaitLatch(latchContainer.countDownLatch4)).isTrue();
 	}
 
+	@Test
+	void shouldRetryFifthTopicWithTwoListenersAndManualAssignment() {
+		logger.debug("Sending two messages to topic " + FIFTH_TOPIC);
```

Comment:
Why do we need this debug message in the test?

Reply:
Well, I find it helpful when debugging tests and set the log level to debug, since there are many moving parts.

Reply:
Is that ok? Please let me know if it isn't. Those tests have a lot going on in parallel with all the retry topics being consumed, so logging is really helpful.